demo/async/CMakeLists.txt

```diff
@@ -7,10 +7,10 @@
 # file was obtained (LICENSE.txt).  A copy of the license may also be
 # found online at https://opensource.org/licenses/MIT.
 
-cmake_minimum_required (VERSION 2.8.7)
+cmake_minimum_required (VERSION 3.13)
```

Comment:
The problem is that NNG_PKGS is not set, so that our nng-config.cmake.in file is not getting populated.  I think this is a regression.  This will be an issue for other libraries as well.

Reply:
Yes, this is a regression introduced with eb328da56c3fc7167b536dcb206df0abb0f4a9b9

Reply:
We need to fix this properly as a result... this PR is a bandaid rather than the correct fix.